.github/workflows/ci-lib-cli-publish.yml

```diff
@@ -32,11 +32,6 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
-      # Checks that the version is not already published and
-      # that there are no uncommitted changes
-      - name: Run prepublish checks
-        run: npm run prepublishOnly
-
```

Comment:
Removes this prepublish checks step now that version bumping is handled by changesets. We removed the same check in the Core library CI previously.